docs/SMG/10-working-with-database.md

```diff
@@ -13,7 +13,9 @@ This will create a `migrations` table in the db, storing a list of all migration
 
 ## Data load
 There currently exists two endpoints on the API for data ingest:
```

Comment:
Can you give instructions on how to call the endpoint, I've briefly seen Chris and Colin try this and they have to set some specifics about form-data etc.

Reply:
Yep, done